src/MCPClient/requirements/base.txt

```diff
@@ -0,0 +1,6 @@
+Django>=1.7,<1.8
+django-mysqlpool==0.1-9
+django-extensions==1.1.1
+MySQL-python==1.2.5
```

Comment:
Can we switch to pymysql instead of this? It supports Python3, and is backwards compatible.  We already use it in agentarchives.